src/accountsdb/db.zig

```diff
@@ -1446,7 +1447,7 @@ pub const AccountsDB = struct {
             const account_file, var account_file_lg = account_file_rw.readWithLock();
             defer account_file_lg.unlock();
 
-            self.logger.infof("cleaning slot: {}...", .{account_file.slot});
+            // self.logger.debugf("cleaning slot: {}...", .{account_file.slot});
```

Comment:
Is this intended to be commented out?

Reply:
Yeah, printing cleaned slots takes up a lot of real estate and isn't hugely useful compared to most of the other logs. If this information becomes valuable during a debugging session we can uncomment it, but otherwise I'd like to leave this commented.